nh_eobs_mental_health/models/nh_eobs_ward_dashboard.py

```diff
@@ -31,7 +31,13 @@ class NHClinicalWardDashboard(orm.Model):
         'capacity_count': fields.integer('Free Beds'),
         'workload_count': fields.integer('All patients associated with ward'),
         'on_ward_count': fields.integer(
-            'All patients in bed and not on Patient Monitoring Exception')
+            'All patients in bed and not on Patient Monitoring Exception'),
+        'refused_obs_count': fields.integer(
+            'All patients currently refusing observations')
+    }
+
+    _defaults = {
+        'refused_obs_count': 0
     }
```

Comment:
Awesome, out of interest what did it return when there was nothing? `None` or `0`?